library/macros/stash-trigger.yaml

```diff
@@ -0,0 +1,90 @@
+- parameter:
```

Comment:
seems like this file is not related to this commit. 
It would be better to move this file to separate commit